lib/rucio/gateway/credential.py

```diff
@@ -57,13 +57,15 @@ def get_signed_url(
     """
 
     kwargs = {'account': account}
-    if not permission.has_permission(issuer=account, vo=vo, action='get_signed_url', kwargs=kwargs, session=session):
-        raise exception.AccessDenied('Account %s can not get signed URL for rse=%s, service=%s, operation=%s, url=%s, lifetime=%s' % (account,
-                                                                                                                                      rse,
-                                                                                                                                      service,
-                                                                                                                                      operation,
-                                                                                                                                      url,
-                                                                                                                                      lifetime))
+    auth_result = permission.has_permission(issuer=account, vo=vo, action='get_signed_url', kwargs=kwargs, session=session)
+    if not auth_result.allowed:
+        raise exception.AccessDenied('Account %s can not get signed URL for rse=%s, service=%s, operation=%s, url=%s, lifetime=%s. %s' % (account,
+                                                                                                                                          rse,
+                                                                                                                                          service,
+                                                                                                                                          operation,
```

Comment:
Is there a reason for this to be across so many lines?

Reply:
I don't think so. It was already formatted like that so I just added to it, but I could reformat it.

Reply:
Good a time as any to do so!

Reply:
Done!